dcicutils/ff_utils.py

```diff
@@ -1010,21 +1010,51 @@ def remove_keys(my_dict, remove_list):
     return store, list(item_uuids)
 
 
-def get_health_page(key=None, ff_env=None):
-    """
-    Simple function to return the json for a FF health page given keys or
-    ff_env. Will return json containing an error rather than raising an
-    exception if this fails, since this function should tolerate failure
-    """
+def _get_page(page='/health', key=None, ff_env=None):
+    """ Wrapper for commonly used code to GET a page from an environment
+        Given keys or ff_env, will return json containing an error rather than raising an
+        exception if this fails, since this function should tolerate failure """
     try:
         auth = get_authentication_with_server(key, ff_env)
-        health_res = authorized_request(auth['server'] + '/health', auth=auth, verb='GET')
+        health_res = authorized_request(auth['server'] + page, auth=auth, verb='GET')
```

Comment:
You're making an assumption about the auth not ending in a slash. Would some sort of join function be better? `os.path.join` isn't quite right, but we might have a utility around to join URL parts, or should make one. (Probably the Python url operations have such a thing and I just don't know it.)